apps/backoffice-v2/src/common/components/atoms/Select_/Select_.tsx

```diff
@@ -6,15 +6,21 @@ import { Select } from '@/common/components/atoms/Select/Select';
 import { FunctionComponent } from 'react';
 import { ISelect_Props } from '@/common/components/atoms/Select_/interfaces';
 
-export const Select_: FunctionComponent<ISelect_Props> = ({ options, ...props }) => {
+export const Select_: FunctionComponent<ISelect_Props> = ({
```

Comment:
`_` suffix since there is already a `Select` in use. This one composes the various components such as `SelectTrigger`, it is a tech debt to replace instances of `Select` with `Select_` and then rename it.

Reply:
Suggestion: give me a name that will express the thing you just mentioned here. `SelectV2`, or rename the old one to `SelectLegacy`